src/content/tools/doc-comment-references.md

```diff
@@ -0,0 +1,125 @@
+linked, if possible, to the documentation page of the element being referenced.
+If the element does not have a documentation page (for example, a function
+parameter, a type variable, or a private class), then no link is created.
+
+## What can be referenced
+
+Here is a quick reference of what can be referenced, and how:
+
+<table>
+  <tr><th>Member</th><th>How to reference</th></tr>
+  <tr>
+    <td>
+Library members (classes, constants, enums, named extensions, extension types, functions, mixins, type aliases)
+    </td>
+    <td>
+Reference with their name, if they are in-scope, prefixed with an import prefix if appropriate.
+
+```dart
+/// [List] is in scope.
+/// So is [math.max].
+```
+    </td>
+  </tr>
+</table>
+
+Most library members can be referenced in a doc comment, including classes,
+constants, enums, named extensions, extension types, functions, mixins, and
+type aliases. This includes all in-scope library members, either declared
+locally, or imported. Library members that are imported with an import prefix
+can be referenced with the prefix. For example:
+
+```dart
+import 'dart:math' as math;
+
+/// [List] is in scope.
+/// So is [math.max].
+int x = 7;
+```
+
+Most members of a class, an enum, an extension, an extension type, and a mixin
+can also be referenced. A reference to a member that is not in scope must be
+qualified (prefixed) with its container's name. For example the `wait` static
+method on the `Future` class can be referenced in a doc comment with
+`[Future.wait]`. This is true for instance members as well; the `add` method
+and the `length` property on the `List` class can be referenced with
+`[List.add]` and `[List.length]`. In-scope members like these can even be
+referenced without the qualifying container name:
+
+```dart
+abstract class MyList<E> implements List<E> {
+  /// Refer to [add] and [contains], which is declared on [Iterable].
+  void myMethod() {}
+}
+```
+
+Unnamed constructors can be referenced by using the `new` name, similar to the
+tear-off of an unnamed constructor. For example, `[DateTime.new]` is a
+reference to the unnamed `DateTime` constructor.
+
+Parameters of a function and parameters of a function type can be referenced in
+a doc comment only when they are in scope. They can therefore only be
+referenced within a doc comment on such a parameter's function or on a type
+alias for such a parameter's enclosing function type.
+
+Type parameters can be referenced in a doc comment only when they are in scope.
+Therefore a type parameter of a method, top-level function, or type alias can
+only be referenced within a doc comment on that element, and a type parameter
+of a class, enum, extension, extension type, and mixin can only be referenced
+within a doc comment on that element or on one of its members.
+
+The doc comment for a type alias that aliases a class, enum, extension type,
+or mixin can't reference any of the aliased type's members, if it has any, as
+if they were in scope.
```

Comment:
I feel the aside of "if it has any" separates the "as if they were in scope" too much. I think it's fine to just drop it:

```suggestion
or mixin can't reference any of the aliased type's members as if they were in scope.
```

Reply:
Yeah good call. Done.